File: src/main/java/org/jukeboxmc/potion/HeroOfTheVillageEffect.java
package org.jukeboxmc.potion;

import org.jetbrains.annotations.NotNull;
import org.jukeboxmc.entity.EntityLiving;

import java.awt.*;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class HeroOfTheVillageEffect extends Effect {

    @Override
    public int getId() {
        return 29;
    }

    @Override
    public @NotNull EffectType getEffectType() {
        return EffectType.HERO_OF_THE_VILLAGE;
    }

    @Override
    public @NotNull Color getEffectColor() {
        return new Color( 68, 255, 68 );
    }

    @Override
    public void apply( EntityLiving entityLiving ) {

    }

    @Override
    public void update( long currentTick ) {

    }

    @Override
    public void remove( EntityLiving entityLiving ) {

    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/effect/HeroOfTheVillageEffect.kt
package org.jukeboxmc.server.effect

import org.jukeboxmc.api.effect.EffectType
import org.jukeboxmc.server.entity.JukeboxEntityLiving
import java.awt.Color

class HeroOfTheVillageEffect : JukeboxEffect() {

    override fun getId(): Int {
        return 29
    }

    override fun getEffectType(): EffectType {
        return EffectType.HERO_OF_THE_VILLAGE
    }

    override fun getEffectColor(): Color {
        return Color(68, 255, 68)
    }

    override fun apply(entityLiving: JukeboxEntityLiving) {

    }

    override fun update(currentTick: Long) {

    }

    override fun remove(entityLiving: JukeboxEntityLiving) {

    }
}